### DISCUSSION TITLE
Setting SMTP when using Docker-Compose

### DISCUSSION THREAD

**Discussion by timowevel1** [ciur/papermerge#481] [General]
Hey,

how to set SMTP credentials when using docker compose? Didnt find any docs about it.

Thanks.

**Comment by ciur** [comment]
Just to make it clear and avoid any confusion: I am speaking about Papermerge 2.1 (in beta phase now).

"SMTP importing" was moved out of core, in other words it will be provided as 3rd party i.e. some sort of comment line utility which will do document importing from SMTP account and upload them to your Papermerge instance (via REST API token).

In case there are volunteers to provide that 3rd party - please stand up - as you have all ingredients for that (REST API works like a charm).
I will provide a small utility of that soon.

Discard my comment if your question was about Papermerge <= 2.0

**Comment by w4tzmann** [comment]
```
In case there are volunteers to provide that 3rd party - please stand up - as you have all ingredients for that (REST API works like a charm).
I will provide a small utility of that soon.
```

I would definitely use the function to import emails. Unfortunately, my programming skills are far from sufficient to write a plugin / program.

I would like to have besides the function to import attachments, also the function to convert the mails themselves into a pdf and process s/mime pgp encrypted mails.

If there is someone who wants to build this and needs a tester: Feel free to contact me, I'm happy to help.
